DiscordBot.DataAccess/MockEventsSheetsService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.Models;
+
+namespace DiscordBot.DataAccess
+{
+    public class MockEventsSheetsService : IEventsSheetsService
```

Comment:
Sorry, I probably wasn't entirely clear when I described creating a mock. I didn't mean a separate class, I meant make the real class implement the methods but with mock implementations for now. Then the only thing you'd need to change later would be to replace the mock implementations. It doesn't even need to keep track of the events or anything, just return something of the correct type 🙂

Reply:
Ah I see! I've simplified it in 729f3d0 as you've suggested, with dummy methods added to the main `EventsSheetsService`, instead of having a fully featured mock events service 🙂